reports/sync.go

```diff
@@ -0,0 +1,258 @@
+package reports
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-github/v57/github"
+	"golang.org/x/exp/slog"
+)
+
+const (
+	vectorUpdateBranch        = "vector-update"
+	gitConfigCredentialHelper = "credential.helper"
+	vectorUpdateCommitMessage = "update test vectors"
+)
+
+// these should be consts but the library expects a pointer to a string, which cannot be done with a const
+var (
+	vectorUpdatePRTitle      = "update test vectors"
+	vectorUpdatePRBody       = "some test vectors were changed or added to the sdk-development repo, so they need to be updated in this repo"
```

Comment:
```suggestion
	vectorUpdatePRBody       = "Some test vectors were changed or added to the sdk-development repo, so they need to be updated in this repo. This is an automated PR that keeps the test vectors in sync from the main test-vectors location: https://github.com/TBD54566975/sdk-development/tree/main/web5-test-vectors"
```

Reply:
I modified this slightly to make use of markdown links. [Sample here](https://github.com/finn-tbd/web5-bf/pull/5).